tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -2480,6 +2500,50 @@ public function userRestoresSpaceObjectsFromTrashRequest(
 		);
 	}
 
+	/**
+	 * @When /^user "([^"]*)" deletes the (?:file|folder) "([^"]*)" from the trash of the space "([^"]*)"$/
+	 * @When /^user "([^"]*)" tries to delete the (?:file|folder) "([^"]*)" from the trash of the space "([^"]*)"$/
```

Comment:
this expression is more readable
```suggestion
	 * @When user :user deletes the file/folder :resource from the trash of the space :spaceName
	 * @When user :user tries to delete the file/folder :resource from the trash of the space :spaceName
```